### PR TITLE
Bug 1873043: Update README.openshift to reflect transition to openshift/kubernetes

### PR DESCRIPTION
**Pull request by marun**

### CODE REVIEW

File: README.openshift.md [hunk 1]
```diff
@@ -1,62 +1,57 @@
 ## Changing API
 
-OpenShift is split into three major repositories:
+OpenShift is split four major repositories:
```
Comment (sttts) [inline README.openshift.md:3]: into four

Comment (sttts) [reply README.openshift.md:3]: OpenShift Kubernetes components. It is not OpenShift as a whole.

Comment (marun) [reply README.openshift.md:3]: Done

File: README.openshift.md [hunk 2]
```diff
@@ -1,62 +1,57 @@
...
+[openshift/library-go](https://github.com/openshift/library-go/) with the changes from step 1, since
+it vendors it.  To do so run `go get github.com/openshift/client-go@master` to pick up the changes
+from step 1 and then run `make update`.  When you're done open a PR against the aforementioned
+repository.
+
+4. The final step happens in the downstream fork of kubernetes, the
+[openshift/kubernetes](https://github.com/openshift/kubernetes/) repository.  As previously, run `go
+get github.com/openshift/library-go@master` to pick up the changes from previous three steps.
+Afterwards, run `make update` to generate the remaining bits in `openshift/kubernetes`
+repository. When you're done open a PR against the aforementioned repository.
```
Comment (sttts) [inline README.openshift.md:35]: Why is this description here at all? How is o/k special as a consumer of library-go? I would expect this merely distributed over api, client-go and library-go repos.

File: README.openshift.md [hunk 3]
```diff
@@ -1,62 +1,57 @@
...
 
-OpenShift is split into three major repositories:
+OpenShift Kubernetes components are split into four major repositories:
 
 1. https://github.com/openshift/api/ - which holds all the external API objects definitions.
-1. https://github.com/openshift/client-go/ - which holds all the client code (written in Go).
-1. https://github.com/openshift/origin/ - which holds the actual code behind OpenShift.
+1. https://github.com/openshift/client-go/ - which holds all the client code (written in Go)
+1. https://github.com/openshift/library-go/ - which holds all the client code (written in Go).
+1. https://github.com/openshift/kubernetes/ - which holds the actual code behind OpenShift.
```
Comment (sttts) [inline README.openshift.md:8]: I still think this block is strange. What are external API objects?

"which holds the actual code behind OpenShift." does not really make sense. Correct: "o/k – the core Kubernetes components with OpenShift specific patches".

File: README.openshift.md (hunk 2 above)
Comment (sttts) [reply README.openshift.md:35]: Would remove the whole list of steps here.

File: README.openshift.md (hunk 3 above)
Comment (marun) [reply README.openshift.md:8]: Done

File: README.openshift.md (hunk 2 above)
Comment (marun) [reply README.openshift.md]: Done

**Pull request merged**